benchmark/README.md

```diff
@@ -0,0 +1,20 @@
+# Benchmarks for Membrane Video Compositor Plugin
+
+## How to run benchmarks:
+
+1.  Enter benchmark folder (cd benchmark)
+2.  Run "mix deps.get" command
+3.  Run command for benchmarks:
+    - benchee benchmarks (run time and memory usage benchmarks): 
+        - for measuring frame composition performance: "mix run lib/benchee/merge_frames.exs"
+        - for measuring raw pipeline performance: "mix run lib/benchee/raw_pipeline.exs"
+        - for measuring h264 pipeline performance: "mix run lib/benchee/h264_pipeline.exs"
+    - beamchmark benchmarks (reductions / contex switches / cpu and memory usage / schedulers)
+        - for measuring raw pipeline performance: "mix run lib/beamchmark/raw_pipeline.exs"
+        - for measuring h264 pipeline performance: "mix run lib/beamchmark/h264_pipeline.exs"
+4. Results will be displayed in console log and saved in html website saved at "results" directory
```

Comment:
Use backticks (triple backticks for console commands) instead of quotation marks. Also, these, instructions, while correct, are a little dry. Maybe don't necessarily use enumeration, use other wording. Take a look at this: https://github.com/membraneframework/membrane_demo/tree/master/rtmp_to_hls